libfreeipmi/cmds/ipmi-lan-cmds.c

```diff
@@ -803,6 +803,335 @@ fiid_template_t tmpl_cmd_get_ip_udp_rmcp_statistics_rs =
     { 0, "", 0}
   };
 
```

Comment:
code cleanup, could you make sure all of these "get_rs" packets are before the "suspend_bmc" packet.  It was sort of out of order and I got lost for a minute.